lib/asn1numericstring.dart

```diff
@@ -0,0 +1,40 @@
+part of asn1lib;
+
+///
+/// An ASN1 Numeric String
+///
+/// A NumericString is a restricted character string type in the ASN.1 notation.
+/// It is used to describe data that does only consist of digits and space character.
+///
+class ASN1NumericString extends ASN1Object {
+  // The decoded string value
+  late String stringValue;
+
+  ///
+  /// Create an [ASN1NumericString] initialized with String value.
+  ///
+  /// Optionally override the tag
+  ///
+  ASN1NumericString(this.stringValue, {int tag = NUMERIC_STRING_TYPE})
```

Comment:
Is it worthwhile doing a quick check to see if the string contains any non numeric/space chars?  I dont think it is required - so its good as is - but would be an option.

Reply:
I've added a simple RegExp based parameter validation and a test for it with a separate commit.